src/renderers/testing/ReactTestMount.js

```diff
@@ -149,24 +149,22 @@ var ReactTestMount = {
 
   render: function(
     nextElement: ReactElement<any>,
-    mockConfig: TestRendererMockConfig,
+    mockConfig?: TestRendererMockConfig,
```

Comment:
Adding `?` at the end of a object/argument key means that the field is optional while adding `?` in the beginning of the type means that it can be null or the type you specified however you always need to pass the argument/key.

```js
/* @flow */

function foo1(a?: { bar: number}) {
}

foo1(); // fine
foo1({}) // not fine

function foo2(a: { bar: ?number }) {
}

foo2({}); // invalid
foo2({ bar: 1}); // fine
foo2({ bar: null}); // fine
```

https://flowtype.org/try/#0PQKgBAAgZgNg9gdzCYAoVUCuA7AxgFwEs5swo44BGACgEMB+ALjAG8wAjWgJ2e0wFt2AUy4BfAJStUo9OSrVxAbjDBgZQtiEYKNFhJVrscfOs2ycBYqTkAmOszaceYen0Eiw+ltNkU7epQMwDQA3WhhCABNtOH8ObmZKCWVVUy1bakcEsD4YGGSgqA0hIA